common/app/views/fragments/amp/googleAnalytics.scala.html

```diff
@@ -26,7 +26,8 @@
                     "contributorIds": "@{content.tags.contributors.map(_.id).mkString(",")}",
                     "keywordIds": "@{content.tags.keywords.map(_.id).mkString(",")}",
                     "toneIds": "@{content.tags.tones.map(_.id).mkString(",")}",
-                    "seriesId": "@{content.tags.series.map(_.id).headOption.getOrElse("")}"
+                    "seriesId": "@{content.tags.series.map(_.id).headOption.getOrElse("")}",
+                    "isHostedFlag": "@{content.tags.isHosted.toString()}"
```

Comment:
Thanks @kelvin-chappell - spoke to Rich and he suggested adding this to `content.metadata` so have done that now